ports/google-cloud-cpp/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "google-cloud-cpp",
-  "version-string": "1.22.0",
+  "version-string": "1.23.0",
```

```suggestion
  "version-semver": "1.23.0",
```

From what I've seen in their GitHub, this library uses SemVer consistently.

---

We do not follow semver, this is documented in each our our libraries:

"Please note that the Google Cloud C++ client libraries do **not** follow Semantic Versioning."

https://github.com/googleapis/google-cloud-cpp/blob/master/google/cloud/pubsub/README.md#google-cloud-pubsub-c-client-library

Furthermore, we strive for API (and **not** ABI) compatibility, and we **do** break the API without increasing the major version number, e.g.:

https://github.com/googleapis/google-cloud-cpp/blob/master/CHANGELOG.md#v1210---2020-12

---

I see, thanks for the clarification! In this case then `version-string` is correct 👍